test/Microsoft.ML.Benchmarks/StochasticDualCoordinateAscentClassifierBench.cs

```diff
@@ -22,7 +21,6 @@ public class StochasticDualCoordinateAscentClassifierBench : WithExtraMetrics
         private readonly string _sentimentDataPath = Program.GetInvariantCultureDataPath("wikipedia-detox-250-line-data.tsv");
         private readonly Consumer _consumer = new Consumer(); // BenchmarkDotNet utility type used to prevent dead code elimination
 
-        private readonly int[] _batchSizes = new int[] { 1, 2, 5 };
```

Comment:
>private readonly int[] _batchSizes = new int[] { 1, 2, 5 }; [](start = 8, length = 59)

`BatchPredictionEngine` is not part of new API, so removing any code related to batch predictions.